### PR TITLE
Add package for enum34 Python module.

### PR DESCRIPTION
**Pull request by fred-wang**

### CODE REVIEW

File: mingw-w64-python-enum34/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Contributor: Frederic Wang <fred.wang@free.fr>
+
+_realname=enum34
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-${_realname}" "${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+pkgver=1.0.4
+pkgrel=1
+pkgdesc='Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4 (mingw-w64)'
```
Comment (Alexpux) [inline mingw-w64-python-enum34/PKGBUILD]: Does we need the for our Python 3.5?

**Comment by fred-wang** [comment]
@Alexpux Right, I've removed the package for Python 3.

**Comment by Alexpux** [comment]
Rename folder with PKGBUILD to match python2 only

**Comment by fred-wang** [comment]
@Alexpux  done

**Pull request merged**